test/ao/sparsity/test_composability.py

```diff
@@ -376,3 +376,43 @@ def test_q_prep_fx_before_s_prep(self):
             sparsity_level, sparse_config[0]["sparsity_level"]
         )
         self.assertGreaterAlmostEqual(cur_sparsity, sparse_config[0]["sparsity_level"])
+
+    def test_s_prep_before_q_prep_fx(self):
```

Comment:
can we add a docstring describing at a high level what we are testing for this test as well

Reply:
yeah, will do